Makefile

```diff
@@ -10,9 +10,9 @@ all: buf
 
 buf: buf-clean
 	@mkdir -p src/gen
-	buf generate buf.build/viamrobotics/goutils --template buf.gen.yaml
-	buf generate buf.build/googleapis/googleapis --template buf.gen.yaml --path google/rpc --path google/api
-	buf generate buf.build/viamrobotics/api --template buf.gen.yaml --path common,component,robot,service,module
+	buf generate buf.build/viamrobotics/goutils --template buf.gen.yaml 
+	buf generate buf.build/googleapis/googleapis --template buf.gen.yaml --path google/rpc --path google/api 
+	buf generate buf.build/viamrobotics/api --template buf.gen.yaml --path common,component,robot,service,module,tagger,app
```

Comment:
Just tried this in python -- you should be able to do `app/v1` as the path, so you only pull in what you need from app and not all the additional fleet stuff